libPhoneNumberTests/generatedJSON/ShortNumberMetadata.json

```diff
@@ -0,0 +1 @@
,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[null,null,"112|999",null,null,null,"112",null,null,[3]],null,[null,null,"1(?:1(?:2|6(?:00[06]|1(?:1[17]|23))|8\\d{2})|9\\d{2})|5[0137]\\d{3}|999",null,null,null,"112"],[null,null,"51\\d{3}",null,null,null,"51012",null,null,[5]],[null,null,"51210",null,null,null,"51210",null,null,[5]],[null,null,"118\\d{2}|5(?:[037]\\d{3}|1210)",null,null,null,"51210",null,null,[5]]],"IL":[null,[null,null,"1\\d{2,4}",null,null,null,null,null,null,[3,4,5]],null,null,[null,null,null,null,null,null,null,null,null,[-1]],[null,null,null,null,null,null,null,null,null,[-1]],null,null,null,"IL",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[null,null,"1(?:0[012]|12)",null,null,null,"112",null,null,[3]],null,[null,null,"1(?:0(?:[012]|400)|1(?:[013-9]\\d|2)|[2-9]\\d{2})",null,null,null,"1455"],[null,null,null,null,null,null,null,null,null,[-1]],[null,null,"10400",null,null,null,"10400",null,null,[5]],[null,null,"10400",null,null,null,"10400",null,null,[5]]],"IM":[null,[null,null,"[189]\\d{2,5}",null,null,null,null,null,null,[3,5,6]],null,null,[null,null,null,null,null,null,null,null,null,[-1]],[null,null,null,null,null,null,null,null,null,[-1]],null,null,null,"IM",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[null,null,"999",null,null,null,"999",null,null,[3]],null,[null,null,"1\\d{2}(?:\\d{3})?|8(?:6444|9887)|999",null,null,null,"150"],[null,null,null,null,null,null,null,null,null,[-1]],[null,null,null,null,null,null,null,null,null,[-1]],[null,null,"8(?:6444|9887)",null,null,null,"86444",null,null,[5]]],"IN":[null,[null,null,"[125]\\d{2,7}",null,null,null,null,null,null,[3,4,5,6,7,8]],null,null,[null,null,"1\\d{2,5}",null,null,null,"105010",null,null,[3,4,5,6]],[null,null,"11[67][0-2]\\d{3}|56161561",null,null,null,"1160530",null,null,[7,8]],null,null,null,"IN",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[null,null,"1(?:0[0128]|12|298)|2611",null,null,null,"108",null,null,[3,4]],null,[null,null,"1(?:0(?:[01248]|5(?:010|6|902)|7(?:[07]|80)|9[0157])|1(?:[289]|[67][0-2]\\d{3})|2(?:1|98)|39|4[01]|55[23]\\d{2}|7000|9(?:0[019]|47|50|6[1347]|[89]))|2611|5(?:14(?:2[5-9]|[34]\\d)|3000|6161(?:17[89]|561)|757555)",null,null,null,"108"],[null,null,"5(?:14(?:2[5-9]|[34]\\d)|757555)",null,null,null,"5757555",null,null,[5,7]],[null,null,"1(?:1(?:[67][0-2]\\d{3}|[89])|21|4[01]|55330|7\\d{3}|9(?:[89]|09))|5(?:3000|6161(?:17[89]|561))",null,null,null,"53000"],[null,null,"1(?:39|90[019])|5(?:14(?:2[5-9]|[34]\\d)|6161(?:17[89]|561)|757555)",null,null,null,"51431",null,null,[3,4,5,7,8]]],"IQ":[null,[null,null,"[1479]\\d{2,4}",null,null,null,null,null,null,[3,4,5]],null,null,[null,null,null,null,null,null,null,null,null,[-1]],[null,null,null,null,null,null,null,null,null,[-1]],null,null,null,"IQ",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[null,null,"1(?:
```

Comment:
Would it be possible to format this into a more readable structure?

https://jsonformatter.org/

Reply:
This would be possible but it wouldn't really be readable anyway. The JSON is generated by libPhoneNumber build tool and it only contains data with pretty much no metadata (as in, it doesn't tell you which field is which), so all you're going to get is something like this which is not really readable :(

```
{  
  "AC":[  
    null,
    [  
      null,
      null,
      "9\\d{2}",
      null,
      null,
      null,
      null,
      null,
      null,
      [  
        3
      ]
    ],
    null,
    null,
    [  
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      [  
        -1
      ]
    ],
    [  
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      [  
        -1
      ]
    ],
    null,
    null,
    null,
    "AC",
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    null,
    [  
      null,
      null,
      "911",
      null,
      null,
      null,
      "911"
    ],
    null,
    [  
      null,
      null,
      "911",
      null,
      null,
      null,
      "911"
    ],
    [  
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      [  
        -1
      ]
    ],
    [  
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      [  
        -1
      ]
    ],
    [  
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      [  
        -1
      ]
    ]
  ]
}
```